### PR TITLE
Done implementing the mobile menu nav

### PR DESCRIPTION
**Pull request by FredrickOtabil**
### Things to checkout for:

- Integrated HMTL + CSS + JS <a href="https://github.com/microverseinc/linters-config/blob/master/html-css-js/README.md" target="_blank">Linters</a>
- Created a directory called 'scripts' and kept all JS files in.
- Edited HTML and CSS files to suit the requirement for this milestone.
- Also added some basic CSS animations and transitions

_Kindly review my pull request for me. All comments and advices are warmly welcomed._

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,11 +17,15 @@
...
-          <i style="font-size: x-large" class="fa-solid fa-bars"></i>
-          <ul>
-            <li><a href="#portfolio">Portfolio</a></li>
-            <li><a href="#about">About</a></li>
-            <li><a href="#contact">Contact</a></li>
+          <i
+            style="font-size: x-large"
+            class="fa-solid fa-bars"
+            id="hamburger"
+          ></i>
```
Comment (m05e5) [inline index.html:24]: - [x] Great job so far. As part of the requirements, kindly display the close `X` icon to close the mobile menu

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @FredrickOtabil , 

Good job so far. !
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] Please check the comments below 

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (FredrickOtabil) [reply index.html:24]: Thank you @m05e5

**Review by pinkmoon25 — APPROVED** [review]
Hi @FredrickOtabil & @Fredrick97-coder , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
GitHub flow :heavy_check_mark: 
Linters :heavy_check_mark: 
Professional Readme :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**